README.md

```diff
@@ -9,8 +9,15 @@ This is experimental!
 GNU make is required.
 
 Run:
-```
-export PATH=/path/to/yk/ykcapi/scripts:${PATH}
+```shell
+export PATH=/path/to/yk/bin:${PATH} # local path to https://github.com/ykjit/yk/blob/master/bin/yk-config
 export YK_BUILD_TYPE=<debug|release>
 make
 ```
+Build artefacts: `./src/lua` and `./src/luac`
+## Runinng
+```shell
```

Comment:
Needs a newline after the heading.

Reply:
added :point_right: e73fedee7e81832d0b3e0ddb26eb58c73ea364fb